index.html

```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
                 <li><img src="pictures/Vector2.png" alt=""></li>
                 <li><img src="pictures/Vector3.png" alt=""></li>
                 <li><img src="pictures/Vector4.png" alt=""></li>
-
             </ul>
-            <!-- <img class="grop" src="pictures/Group.png" alt=""> -->
         </section>
-
-
     </section>
+    <section class="second-main">
+        <h4>My Recent Works</h4>
+        <img src="pictures/line.png" class="line" alt="">
+        <img src="pictures/ImgPlaceholder.png" alt="">
+        <h5>Multi-Post Stories</h5>
+        <p class="par-black">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div>
+            <img src="pictures/Tag.png" alt="">
+            <img src="pictures/Tag(1).png" alt="">
+            <img src="pictures/Tag(2).png" alt="">
+            <img src="pictures/Tag(3).png" alt="">
+        </div>
+        <img src="pictures/Enable.png" class="fiveg" alt="">
+        <section class="activ-g">
+            <h6 class="en">Profesional Art<br>Printing Data <br></h6>
+            <p class="text2">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups
+                required. has been the <br>
+                industry's standard</p>
+            <div class="en">
+                <img src="pictures/translu.png" alt="">
+                <img src="pictures/translu1.png" alt="">
+                <img src="pictures/translu2.png" alt="">
+            </div>
```

Comment:
- [ ] We are still not using proper `<ul>` and `<li>` elements to list the technologies here. Please fix this by using the proper list tags in this and the other projects.